Core/Object Arts/Dolphin/IDE/Base/SmalltalkWorkspace.cls

```diff
@@ -66,7 +66,6 @@ applyOptions
 	view autoCompletionSeparator: $\x7F.
 	self areVariableTipsEnabled ifTrue: [view isBackgroundDwellEnabled: true].
 	"#2137"
-	view sciSetCodePage: KernelLibrary default getACP.
```

Comment:
Not part of the upgrade. This should have been removed with the switch to UTF-8, and may have been causing problems for anyone using a code page that Scintilla recognised.